Sources/StytchCore/StytchB2BClient/Models/MemberSession.swift

```diff
@@ -0,0 +1,16 @@
+import Foundation
+
+public struct MemberSession: Codable {
+    public typealias ID = Identifier<Self, String>
+
+    public var id: ID { memberSessionId }
+    public let organizationId: Organization.ID
+    public let memberId: Member.ID
+    public let startedAt: Date
+    public let lastAccessedAt: Date
+    public let expiresAt: Date
+    public let authenticationFactors: [AuthenticationFactor]
+    public let customClaims: JSON?
```

Comment:
If it were serialized as json wouldn't it be valid JSON? or am I missing something? I'm essentially saying this will be a typed JSON blob, which you need to fish around in and get what you expect or need from.

Reply:
Oh, not a question of if it can be JSON (it's coming as JSON down the wire!), just more a question of if Android is doing casting work it doesn't need to, and is raw JSON acceptable for the consumer or do they want it in a more easily digestible format?

Reply:
Ahh, got it. this JSON type isn't exactly a JSON string. it's an enum with cases for each of the possible JSON types, with convenience accessors, so it should be reasonable to work with. `customClaims["some_key"].stringValue` will allow them to grab some string they expect, while they can otherwise dig further in `customClaims["key1"]["key2"][0].numberValue` will give you back an optional number.

I think it's actually more convenient than me doing something like `[String: Any]` on my end, as they would have to do all the intermediary casting to get what they wanted out of it: `((customClaims["key1"] as? [String: Any])?["key2"] as? [Double])?[0]`